docker-build/slave_docker_run.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+# Run the slave docker with path one level down as root dir.
+#
+set -x
+
+docker run --rm=true --privileged --init \
+    -v "${PWD}:/lom-root" \
+    -v "/tmp/docklock:/tmp/docklock"\
+    -w "/lom-root" -e "http_proxy=" -e "https_proxy=" -e "no_proxy=" -it \
+    lom-slave-bullseye-admin:1234
```

Comment:
Just one slave container only.
The LoM container we build will be distinct by image name  & will be versioned too and it will not be loaded in your Dev VM by default